.env.example

```
@@ -47,4 +47,12 @@ DB_PORT=5432
 # REVIEW_DRIVE_EMAIL=
-# REVIEW_DRIVE_PASSWORD=
\ No newline at end of file
+# REVIEW_DRIVE_PASSWORD=
+
+# =============================================================================
+# OPTIONAL - For Trending Analytics (DynamoDB)
+# =============================================================================
+# AWS_ANALYTICS_ACCESS_KEY_ID=
+# AWS_ANALYTICS_SECRET_ACCESS_KEY=
+# AWS_REGION=us-east-1
+# DYNAMODB_TABLE_NAME=trending_analytics
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)